examples/amr_human_sample_paired_rds_dag.json

```diff
@@ -0,0 +1,26 @@
+{
+  "output_dir_s3": "s3://idseq-samples-prod/samples/12/5815/postprocess_test",
```

Comment:
Why use `s3://idseq-samples-prod/samples/12/5815/postprocess_test` when the input is from `s3://idseq-samples-prod/samples/64/4241/fastqs`? In that case, it would make more sense to put the output in `s3://idseq-samples-prod/samples/64/4241/postprocess_test`

Reply:
resolved